tests/Feature/Conversations/Messages/IndexTest.php

```diff
@@ -19,6 +19,18 @@ public function user_can_index_messages_of_a_conversation(): void
         $this->assertResponse($response);
     }
 
+    /** @test */
+    public function user_can_index_and_mark_messages_as_read(): void
```

Comment:
How are you testing the marking as read during this test? This test is the same as your "normal" test without any assertions.

Reply:
Ah I see it in snapshot. Maybe mention this.